src/LevelDBStore/Plugins/Storage/Store.cs

```diff
@@ -11,21 +11,23 @@ internal class Store : IStore
     {
         private const byte SYS_Version = 0xf0;
         private readonly DB db;
+        private bool readCache;
 
-        public Store(string path)
+        public Store(string path, bool readCache)
         {
             this.db = DB.Open(path, new Options { CreateIfMissing = true });
             byte[] value = db.Get(ReadOptions.Default, Helper.CreateKey(SYS_Version));
             if (value != null && Version.TryParse(Encoding.ASCII.GetString(value), out Version version) && version >= Version.Parse("3.0.0"))
                 return;
 
             WriteBatch batch = new WriteBatch();
+            this.readCache = readCache;
 
             if (value != null)
             {
                 // Clean all entries only if the version are different
 
-                ReadOptions options = new ReadOptions { FillCache = false };
+                ReadOptions options = new ReadOptions { FillCache = readCache };
```

Comment:
The data are to be deleted, so we don't need to cache them.

Reply:
Not only deleted items. Upon UpdateCurrentSnapshot after persisting block, all data cached will be flushed due to creating a new snapshot.

Reply:
Can you see the code again?

Reply:
It seems all cached data are refreshed after UpdateCurrentSnapshot:

Blockchain.UpdateCurrentSnapshot
![image](https://user-images.githubusercontent.com/43407364/83593812-3d449580-a590-11ea-9060-3e1d723bed99.png)

Blockchain.GetSnapshot
![image](https://user-images.githubusercontent.com/43407364/83593861-56e5dd00-a590-11ea-8187-42115b62c8fd.png)

SnapshotView constructor
![image](https://user-images.githubusercontent.com/43407364/83593905-6ebd6100-a590-11ea-8c8b-3387b0ae60d3.png)

LevelDB Store.GetSnapshot
![image](https://user-images.githubusercontent.com/43407364/83594120-f7d49800-a590-11ea-9af4-b9a2b237d2c3.png)

LevelDB Snapshot constructor
![image](https://user-images.githubusercontent.com/43407364/83594194-2f434480-a591-11ea-86a7-c6f702bf9214.png)

DB.GetSnapshot
![image](https://user-images.githubusercontent.com/43407364/83594235-4a15b900-a591-11ea-8fdc-74e4b3372c2f.png)

Snapshot constructor
![image](https://user-images.githubusercontent.com/43407364/83594262-5d288900-a591-11ea-8940-dbe819e9d95a.png)

Reply:
https://github.com/neo-project/neo-modules/blob/eb75e0d730ebe0e62eafd22b2b8859377936bca4/src/LevelDBStore/Plugins/Storage/Store.cs#L28-L35

See it again.

Reply:
Sorry, I misunderstood, here don't need to assign the fill cache value..